packages/cli/src/services/frontend.service.ts

```diff
@@ -115,9 +115,9 @@ export class FrontendService {
 				apiHost: config.getEnv('diagnostics.config.posthog.apiHost'),
 				apiKey: config.getEnv('diagnostics.config.posthog.apiKey'),
 				autocapture: false,
-				disableSessionRecording: config.getEnv(
-					'diagnostics.config.posthog.disableSessionRecording',
-				),
+				disableSessionRecording:
+					config.getEnv('diagnostics.config.posthog.disableSessionRecording') ||
```

Comment:
why even keep the `disableSessionRecording` flag then? should we just remove it completely, and just use the deployment type?

Reply:
Excellent point. I think we could just check the domain on FE and disable it based on that